app/bundles/PluginBundle/Translations/en_US/messages.ini

```diff
@@ -91,3 +91,4 @@ mautic.plugin.integration.contact.timeline.link="Contact's timeline link"
 mautic.plugin.integration.campaigns="Push contacts to this integration campaign"
 mautic.plugin.config.campaign.member.chooseone="Choose a campaign to insert contacts into"
 mautic.plugin.integration.campaigns.member.status="Campaign member status"
+mautic.plugin.integration.contact.donotcontact="Do not contact"
```

Comment:
Let's change this to `mautic.plugin.integration.contact.donotcontact.email="Do not contact by email"` since this is currently specifically to email channel.